spec/plugman/install.spec.js

```diff
@@ -115,7 +115,12 @@ describe('plugman/install', () => {
         fetchSpy = jasmine.createSpy('plugmanFetch').and.returnValue(Promise.resolve(pluginDir('com.cordova.engine')));
         install.__set__({ plugmanFetch: fetchSpy });
 
-        spyOn(superspawn, 'spawn').and.returnValue(Promise.resolve(''));
+        execaSpy = jasmine.createSpy('execa');
+        execaSpy.and.returnValue(Promise.resolve({ stdout: '' }));
+        install.__set__('execa', execaSpy);
+
+        execaSpy.and.returnValue(Promise.resolve({ stdout: '' }));
```

Comment:
Why are we doing this twice?

Reply:
Rebase issue? Not sure why it is there twice.